bolt-modules/boltlib/lib/puppet/functions/run_command.rb

```diff
@@ -26,7 +26,7 @@
   # @param options Additional options: '_catch_errors', '_run_as'.
   # @return A list of results, one entry per target.
   # @example Run a command on targets
-  #   run_command('hostname', $targets, '_catch_errors' => true)
+  #   run_command('hostname', $targets, 'Get hostname')
```

Comment:
Not sure I follow why this changed?

Reply:
The example didn't map to the function, which takes a 3rd `description` argument before options.